src/lib/mongoBackend/MongoCommonUpdate.cpp

```diff
@@ -2454,6 +2460,285 @@ static bool forwardsPending(UpdateContextResponse* upcrsP)
   return false;
 }
 
+/* ****************************************************************************
+*
+* updateEntity -
+*/
+static void updateEntity
```

Comment:
Always good to get smaller functions, however, the PR review gets more tricky, at least if there are changes inside the moved code.  Are there any changes in updateEntity vs what we had before?